Demo/Demo/SwiftUIComponents/ErrorView.swift

```diff
@@ -0,0 +1,17 @@
+import SwiftUI
+
+struct ErrorView: View {
+
+    let errorText: String
+
+    var body: some View {
+        VStack {
+			Text("\(errorText)")
```

Comment:
It looks like this `Text` may be indented too far

Reply:
addressed in commit 121c4eadb22995e34dbf68e2aee497baa92e2249